kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java

```diff
@@ -808,8 +808,8 @@ public Watch watch(ListOptions options, final Watcher<T> watcher) {
     } catch (MalformedURLException e) {
       throw KubernetesClientException.launderThrowable(forOperationType("watch"), e);
     } catch (KubernetesClientException ke) {
-
-      if (ke.getCode() != 200) {
+      List<Integer> furtherProcessedCodes = Arrays.asList(200, 503);
```

Comment:
Maybe we can move this to a static concurrent set

Reply:
Why static concurrent set? This is just more readable version of `ke.getCode() != 200 && ke.getCode() != 503`. It doesn't have any deeper purpose.

Reply:
Maybe a concurrent set is a bit overkilling in this case.

Reply:
It just doesn't seem optimal to create a new instance of a List each time the method gets called especially if the list is never changed.
It doesn't make sense either to make it concurrent since the list is never written to (besides when it's initialized).

Anyway, it's fine as it is. :heavy_check_mark: